js/components/pastEvents/PastEventsDataFetcher.js

```diff
@@ -12,30 +14,62 @@ const PastEventsDataFetcher = () => {
   const [currentPage, setCurrentPage] = useState(1)
   const [isFetchingMore, setIsFetchingMore] = useState(false)
 
+  const [checkedWorkingGroups, setCheckedWorkingGroups] = useState({})
+  const [checkedTypes, setCheckedTypes] = useState({})
+
+  const [dataWithFilters, setDataWithFilters] = useState([])
+
   const fetchingData = (page) => {
-    fetch(`https://newsroom.eclipse.org/api/events?page=${page}&pagesize=4`)
-      .then((res) => res.json())
-      .then(
-        (result) => {
-          setLoading(false)
-          setIsFetchingMore(false)
-          setEvents([...events, ...result.events])
-        },
-        // Note: it's important to handle errors here
-        // instead of a catch() block so that we don't swallow
-        // exceptions from actual bugs in components.
-        (error) => {
-          setLoading(false)
-          setIsFetchingMore(false)
-          setError(error)
-        }
-      )
+    let url = `https://newsroom.eclipse.org/api/events?page=${page}&pagesize=6`
+    fetch(url)
+    .then((res) => res.json())
+    .then(
+      (result) => {
+        setLoading(false)
+        setIsFetchingMore(false)
+        setEvents([...events, ...result.events])
+      },
+      // Note: it's important to handle errors here
+      // instead of a catch() block so that we don't swallow
+      // exceptions from actual bugs in components.
+      (error) => {
+        setLoading(false)
+        setIsFetchingMore(false)
+        setError(error)
+      }
+    )
+  }
+
+  const fetchingDataWithParas = (page, para) => {
+    let url = `https://newsroom.eclipse.org/api/events?page=${page}&pagesize=6&parameters[publish_to]=${para}`
```

Comment:
You can have more than 1 publish_to value:

Example:
https://newsroom.eclipse.org/api/events?page=1&pagesize=6&parameters[publish_to][]=ecd_tools&parameters[publish_to][]=jakarta_ee